src/components/DeFi/components/MaxButtonGroup.tsx

```diff
@@ -0,0 +1,30 @@
+import { Button, ButtonGroup } from '@chakra-ui/react'
+import { Amount } from 'components/Amount/Amount'
+
+type MaxButtonGroupProps = {
+  options: number[]
+  value?: number | null
+  onClick: (args: number) => void
```

Comment:
Better parameter naming? Or is this one supposed to be generic?

Reply:
This one is generic, as it only has one `onClick` can't be confused on what is being clicked on.